.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         include:
           - build: pinned
             os: ubuntu-20.04
-            rust: 1.72
+            rust: 1.75
```

Comment:
note: I went ahead and bumped Rust forward to 1.75. This is because Axum Test moved to use async trait fn, which came into being in 1.75.